ocs_ci/utility/framework/base_initializer.py

```
@@ -0,0 +1,205 @@
+        cluster_path = os.path.expanduser(args.cluster_path)
+
+        if not os.path.exists(cluster_path):
+            raise FileNotFoundError(f"No such directory: {cluster_path}")
+        else:
+            config.ENV_DATA["cluster_path"] = cluster_path
+
+        if not cluster_name:
+            try:
+                config.ENV_DATA["cluster_name"] = get_cluster_name(cluster_path)
+            except FileNotFoundError:
+                raise ClusterNameNotProvidedError()
```

Ruff (0.15.15)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)